core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-edit-ema-workflow-actions/dot-edit-ema-workflow-actions.component.ts

```diff
@@ -0,0 +1,179 @@
+import { AsyncPipe } from '@angular/common';
+import {
+    Component,
+    EventEmitter,
+    Input,
+    OnChanges,
+    Output,
+    SimpleChanges,
+    inject,
+    signal
+} from '@angular/core';
+
+import { MessageService } from 'primeng/api';
+import { ButtonModule } from 'primeng/button';
+
+import { catchError, map, take } from 'rxjs/operators';
+
+import {
+    DotHttpErrorManagerService,
+    DotMessageService,
+    DotWizardService,
+    DotWorkflowActionsFireService,
+    DotWorkflowsActionsService,
+    DotWorkflowEventHandlerService
+} from '@dotcms/data-access';
+import { DotCMSContentlet, DotCMSWorkflowAction, DotWorkflowPayload } from '@dotcms/dotcms-models';
+import { DotMessagePipe, DotWorkflowActionsComponent } from '@dotcms/ui';
+
+@Component({
+    selector: 'dot-edit-ema-workflow-actions',
+    standalone: true,
+    imports: [AsyncPipe, DotWorkflowActionsComponent, ButtonModule, DotMessagePipe],
+    providers: [
+        DotWorkflowActionsFireService,
+        DotWorkflowEventHandlerService,
+        DotWorkflowsActionsService,
+        DotHttpErrorManagerService
```

Comment:
If you are injecting this here, why are you also doing it in the shell component?

Reply:
Forgot to remove them. Nice catch @fmontes!